livre/eq/Config.cpp

```diff
@@ -194,16 +199,22 @@ bool Config::init( const int argc, char** argv )
 
 uint32_t Config::frame()
 {
+    if( _impl->dataFrameRange == INVALID_FRAME_RANGE )
+        return eq::Config::finishFrame();
```

Comment:
It's definitely untested, and unusual. start/finishFrame are meant to be called in pairs.

Why do you call it here?